Makefile

```diff
@@ -424,6 +430,37 @@ helm-install-advanced-local-context: manifests
 		--set enablePodLevel=true \
 		--set enableAnnotations=true
 
+helm-install-hubble:
+	helm upgrade --install retina ./deploy/hubble/manifests/controller/helm/retina/ \
+		--namespace kube-system \
+		--set operator.enabled=true \
+		--set operator.repository=$(IMAGE_REGISTRY)/retina-operator-enterprise \
+		--set operator.tag=$(HELM_IMAGE_TAG) \
+		--set agent.enabled=true \
+		--set agent.repository=$(IMAGE_REGISTRY)/retina-agent-enterprise \
+		--set agent.tag=$(HELM_IMAGE_TAG) \
+		--set agent.init.enabled=true \
+		--set agent.init.repository=$(IMAGE_REGISTRY)/retina-agent-init-enterprise \
+		--set agent.init.tag=$(HELM_IMAGE_TAG) \
+		--set logLevel=info \
+		--set hubble.tls.enabled=$(ENABLE_TLS) \
+		--set hubble.relay.tls.server.enabled=$(ENABLE_TLS) \
+		--set hubble.tls.auto.enabled=$(ENABLE_TLS) \
+		--set hubble.tls.auto.method=cronJob \
+		--set hubble.tls.auto.certValidityDuration=1 \
+		--set hubble.tls.auto.schedule="*/10 * * * *"	
+
+.PHONY: deploy-operator
+deploy-operator-hubble:
```

Comment:
This is not required. Operator will be deployed as part of retina install.

Reply:
removed